lib/nexpose/scan.rb

```diff
@@ -407,9 +407,10 @@ def import_scan(site_id, zip_file)
       data = Rex::MIME::Message.new
       data.add_part(site_id.to_s, nil, nil, 'form-data; name="siteid"')
       data.add_part(session_id, nil, nil, 'form-data; name="nexposeCCSessionID"')
-      scan = ::File.new(zip_file, 'rb')
-      data.add_part(scan.read, 'application/zip', 'binary',
-                    "form-data; name=\"scan\"; filename=\"#{zip_file}\"")
+      ::File.new(zip_file, 'rb') do |scan|
```

Comment:
Ah I thought File.new and File.open were actually the same method (aliases). I'll change it.